src/main/java/kr/codesqaud/cafe/post/controller/PostController.java

```diff
@@ -0,0 +1,53 @@
+package kr.codesqaud.cafe.post.controller;
+
+import kr.codesqaud.cafe.post.controller.request.PostWriteRequest;
+import kr.codesqaud.cafe.post.controller.response.PostDetailResponse;
+import kr.codesqaud.cafe.post.controller.response.PostListResponse;
+import kr.codesqaud.cafe.post.service.PostService;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.util.List;
+
+@Controller
+@RequestMapping("/posts")
+public class PostController {
+
+    private final PostService postService;
+
+    public PostController(PostService postService) {
+        this.postService = postService;
+    }
+
+    @GetMapping("/write")
+    public String writePost() {
+        return "post/form";
+    }
+
+    @PostMapping("/write")
```

Comment:
REST API 를 작성하실때 **행위(Action)** 보다는 **자원(Resource)** 위주로 작성해주시는것이 좋습니다.
```suggestion
    @PostMapping("/posts")
```